Makefile

```diff
@@ -2028,6 +2028,9 @@ openstack_kuttl_run: ## runs kuttl tests for the openstack operator, assumes tha
 .PHONY: openstack_kuttl
 openstack_kuttl: export NAMESPACE = ${OPENSTACK_KUTTL_NAMESPACE}
 openstack_kuttl: input deploy_cleanup openstack openstack_deploy_prep ## runs kuttl tests for the openstack operator. Installs openstack operator and cleans up previous deployments before running the tests, cleans up after running the tests.
+	# Wait until OLM installs openstack CRDs
+	timeout $(TIMEOUT) bash -c "while ! (oc get crd | grep openstack.org); do sleep 1; done"
+	bash -c 'oc get crd openstacks.operator.openstack.org && make openstack_init'
```

Comment:
Won't the call to `make openstack_init` be a problem here until [1] merges?  `openstack_init` assumes that the new initialization sample config is available in the OpenStack operator repo [2].  I think that would throw an error until [1] merges.

[1] https://github.com/openstack-k8s-operators/openstack-operator/pull/1185
[2] https://github.com/openstack-k8s-operators/install_yamls/blob/b6028e38fd1ff8daab363a63f710f4fbed3d1533/Makefile#L746-L747